data/json/items/melee/swords_and_blades.json

```diff
@@ -1645,7 +1646,7 @@
     "weight": "1150 g",
     "bashing": 3,
     "cutting": 14,
-    "to_hit": { "grip": "weapon", "length": "hand", "surface": "any", "balance": "neutral" },
+    "to_hit": { "grip": "weapon", "length": "hand", "surface": "point", "balance": "good" },
```

Comment:
Wow, the recipe is super dumb. 4 planks and 5 rebar and nothing to actually strap it. I put it at weapon because that's what strapped is supposed to be but yeah for this I might even go with none unless the recipe gets changed.